### ISSUE TITLE
Mac SDK fails to open with error -10810

### ISSUE THREAD

**Issue by Thunderforge** [jMonkeyEngine/sdk#213]
I'm trying to install JMonkeyEngine SDK (3.2.4-stable) for the first time on my Mac. I downloaded the latest Mac version from [here](https://github.com/jMonkeyEngine/sdk/releases) . I unpacked the .tgz and got "jMonkeyEngine SDK Installer". But when I try to open it, it immediately shuts down. If I try to open it in the Terminal, I get the following:

```
$ open -a jMonkeyEngine\ SDK\ Installer.app 
LSOpenURLsWithRole() failed for the application jMonkeyEngine SDK Installer.app with error -10810.
```

According to [this web popst](https://apple.stackexchange.com/questions/181026/lsopenurlswithrole-failed-with-error-10810-cant-open-install-os-x-yosemite), the issue is either the executable flag is not set (fixable with `chmod +x`) or it has been quarantined. I tried both fixes suggested, but couldn't get it to work.

I'm running macOS 10.14.6 and have my Gatekeeper set to allow apps from "App Store and identified developers". I previously opened the installer by right-clicking on the SDK and selecting "Open", then saying that I accept that it's not from a trusted developer.

**Comment by edward3h** [comment]
I'm seeing the same error. Tried downloading 3.2.4-stable, 3.2.1-stable and one in between (sorry I lost track). I get the same error with all of them.

**Comment by MeFisto94** [comment]
Sorry for the very late reply.
We can only hope this is solved with the 3.3 SDK release, which is still pending, because the error code basically means "unknown error" and it started appearing with some Mac OS Update.
As a workaround you can try downloading the platform independent release `jmonkeyplatform.zip` and execute bin/netbeans.

This does not bundle any jdk, is no installer and does not bundle blender, but it's more simple and thus could work.

**Comment by MeFisto94** [comment]
Good news, it seems like this is solved in 3.3-beta1-sdk1,
@swiftech @edward3h @Thunderforge re-testing would be really appreciated :)

**Comment by MeFisto94** [comment]
No feedback so far, thus I'm closing it.
The issue can be re-opened if the new netbeans version does NOT fix it.
